Core/libMOOS/Comms/MOOSCommClient.cpp

```diff
@@ -977,6 +978,7 @@ bool CMOOSCommClient::ConnectToServer()
 		//suggestion to move this to here accpted by PMN on 21st Jan 2009
         //we must be connected for user callback to work..
         m_bConnected = true;
+        m_dfLastConnectionTime = MOOS::Time();
```

Comment:
preferably this would be MOOSLocalTime() to avoid any chance of strange behaviour around accelerated time because we actually want a real-world timestamp here